src/tests/Microsoft.PowerFx.Connectors.Tests/MergeRecordTests.cs

```diff
@@ -106,7 +106,7 @@ public void MergeRecords_Invalid()
             RecordValue rv2 = engine.Eval(@"{a:""x""}") as RecordValue;            
 
             var ex = Assert.Throws<ArgumentException>(() => ArgumentMapper.MergeRecords(rv1, rv2));
-            Assert.Equal("Cannot merge a of type NumberValue with a of type StringValue", ex.Message);
+            Assert.Equal("Cannot merge a of type DecimalValue with a of type StringValue", ex.Message);
```

Comment:
>a of [](http://example.com/codeflow?start=39&length=4)

Not your change, but this message is weird.

Reply:
"a" is the name of the field, so it should be wrapped in single quotes or something, but not horrible if you see the whole context.